Formula/pstoedit.rb

```diff
@@ -27,8 +27,7 @@ class Pstoedit < Formula
   fails_with gcc: "6"
 
   def install
-    ENV.cxx11
-
+    on_macos { ENV.cxx11 }
```

Comment:
Can we leave this out on macOS too?

Reply:
It fails on configure for macOS:
```
checking whether we are using the GNU C++ compiler... yes
checking whether clang++ accepts -g... yes
checking dependency style of clang++... none
checking for g++ std... configure: error: your version of g++ seems to be too old.
```

configure.ac:
```
AC_MSG_CHECKING(for g++ std)
AC_COMPILE_IFELSE([AC_LANG_SOURCE([[ constexpr int ce = 10; int * p = nullptr; ]])],
               [AC_MSG_RESULT(you seem to have suitable version of g++)],
               [AC_MSG_ERROR([your version of g++ seems to be too old.])])
```